egs/csj/ASR/RESULTS.md

```diff
@@ -0,0 +1,212 @@
+# Results
+
+## Streaming Zipformer-Transducer (Pruned Stateless Transducer + Streaming Zipformer)
+
+### [pruned_transducer_stateless7_streaming](./pruned_transducer_stateless7_streaming)
+
+See <https://github.com/k2-fsa/icefall/pull/892> for more details.
+
+You can find a pretrained model, training logs, decoding logs, and decoding results at:
+<https://huggingface.co/TeoWenShen/icefall-asr-csj-pruned-transducer-stateless7-streaming-230208>
+
+Number of model parameters: 75688409, i.e. 75.7M.
+
+#### training on disfluent transcript
+
+The CERs are:
+
+| decoding method	         | chunk size | eval1 | eval2 | eval3 | excluded | valid | average     | decoding mode       |
+| -------------------------- | ---------- | ----- | ----- | ----- | -------- | ----- | ------------------- | ------------------- |
+| fast beam search           | 320ms	  | 6.27  | 5.13  | 5.05  | 6.30     | 5.42  | --epoch 30 --avg 14 | simulated streaming |
+| fast beam search           | 320ms	  | 5.91  | 4.30  | 4.53  | 6.13     | 5.13  | --epoch 30 --avg 14 | chunk-wise          |
+| greedy search              | 320ms      | 5.81  | 4.29  | 4.63  | 6.07     | 5.13  | --epoch 30 --avg 14 | simulated streaming |
+| greedy search              | 320ms      | 5.91  | 4.50  | 4.65  | 6.36     | 5.34  | --epoch 30 --avg 14 | chunk-wise          |
+| modified beam search       | 320ms      | 5.59  | 4.20  | 4.39  | 5.54     | 4.90  | --epoch 30 --avg 14 | simulated streaming |
+| modified beam search       | 320ms      | 5.79  | 4.48  | 4.41  | 5.98     | 5.19  | --epoch 30 --avg 14 | chunk-wise          |
+| fast beam search           | 640ms      | 5.76  | 4.35  | 4.39  | 5.40     | 4.92  | --epoch 30 --avg 14 | simulated streaming |
+| fast beam search           | 640ms      | 5.45  | 4.31  | 4.29  | 5.61     | 4.97  | --epoch 30 --avg 14 | chunk-wise          |
+| greedy search              | 640ms      | 5.37  | 3.94  | 4.03  | 5.22     | 4.77  | --epoch 30 --avg 14 | simulated streaming |
+| greedy search              | 640ms      | 5.77  | 4.44  | 4.49  | 5.70     | 5.29  | --epoch 30 --avg 14 | chunk-wise          |
+| modified beam search       | 640ms      | 5.19  | 3.81  | 3.93  | 4.83     | 4.59  | --epoch 30 --avg 14 | simulated streaming |
+| modified beam search       | 640ms      | 6.71  | 5.35  | 4.95  | 6.06     | 5.94  | --epoch 30 --avg 14 | chunk-wise          |
+
+Note: `simulated streaming` indicates feeding full utterance during decoding using `decode.py`,
+while `chunk-size` indicates feeding certain number of frames at each time using `streaming_decode.py`.
+
+The training command was:
+```bash
+./pruned_transducer_stateless7_streaming/train.py \
+  --feedforward-dims  "1024,1024,2048,2048,1024" \
+  --context-size 2 \
+  --world-size 8 \
+  --num-epochs 30 \
```

Comment:
Are the results listed in the table from the best combination `--epoch 30 --avg 14`? That is, have you tried other combinations of `--epoch --avg`?

Reply:
Yes, the results listed in the table are from an exhaustive search for the best combination, for both disfluent and fluent.